HW_5-6.md

```diff
@@ -0,0 +1,27 @@
+### Awk /ɔːk/
+#### 1. What is the most frequent browser?
+```
+awk '{print $12}' access.log | sort | uniq -c | sort -rn
```

Comment:
You should take in account the entire field User-Agent. Mozilla is a company, isn`t it?